app/Controllers/subscriptionController.php

```diff
@@ -123,6 +124,9 @@ public function feedAction() {
 			if ($cookie != '') {
 				$opts[CURLOPT_COOKIE] = $cookie;
 			}
+			if ($useragent != '') {
```

Comment:
Please, use `!==` instead of `!=`.

Reply:
I know that the code around does not use it, but I think it should. We will fix it when working on it.